lib/chiscore/checkins.rb

```diff
@@ -42,7 +42,8 @@ def times_for(checkpoint)
       end
 
       def checkout(checkpoint, team, admin)
-        raise EarlyCheckout if !admin && early?(team)
```

Comment:
From my searching it looks like the `early?` method can now be removed as well.

Reply:
Confirmed. I'll make a change there. I think there are a few pieces of this Redis code that are confusing, and might need a bit of cleanup.

Per Liz's suggestion I have a few things that we can work on early this year now that the dust is settling (and hopefully that will make our lives easier pre-race next year) -- you wanna get a drink or something in the next week or two to talk shop?

Reply:
(I'll just remove `early?` for now but I think some of these things are confusingly named and I think the administrative use-cases we want to add might drive out some other design improvements)

Reply:
Sounds good; I'll follow up with a gcal invite 😉